### PR TITLE
USHIFT-1804: Report OK status for IP and SSH in offline scenarios

### PR DESCRIPTION
**Pull request by ggiguash**
A follow-up fix on #2518

### CODE REVIEW

**Comment by pmtk** [comment]
/lgtm
/hold
Unhold when ready

**Comment by ggiguash** [comment]
/unhold 
The remaining problem in the periodic test is about FDO sos-report failure.
It will be fixed in another PR.

File: test/bin/scenario.sh [hunk 1]
```diff
@@ -385,8 +385,9 @@ launch_vm() {
         touch "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${vmname}/ip"
 
         echo "VM ${full_vmname} has no NICs, skipping IP assignment and ssh polling"
-        record_junit "${vmname}" "ip-assignment" "SKIPPED"
-        record_junit "${vmname}" "ssh-access" "SKIPPED"
+        # Anything other than "OK" status is reported as an error
```
Comment (dhellmann) [inline test/bin/scenario.sh:388]: That could be fixed by fixing the logic in record_junit.

Comment (dhellmann) [reply test/bin/scenario.sh:388]: See #2556

**Pull request merged**